support-frontend/assets/components/headers/header/header.scss

```diff
@@ -213,3 +219,103 @@ $header-links-min-width: gu-span(12);
     }
   }
 }
+
+.component-header--display-checkout {
+  .component-header__wrapper {
+    max-width: 50rem;
+
+    @include mq($from: tablet, $until: desktop) {
+      max-width: 100%;
+      padding: 0;
+      margin: 0;
+    }
+
+    @include mq($from: leftCol) {
+      max-width: 60rem;
```

Comment:
try to use `gu-span(*)` here rather than a fixed pixel size

Reply:
Again here - I'm not able to get the lines in the right place with columns. Any suggestions as to how to make this work?